sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -240,6 +240,23 @@ public partial class CharFilter
         internal CharFilter() { }
         public string Name { get { throw null; } set { } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct CharFilterName : System.IEquatable<Azure.Search.Documents.Indexes.Models.CharFilterName>
+    {
+        private readonly object _dummy;
```

Comment:
1. Why are privates showing up?
2. We shouldn't be using "dummy" in anything, or are these compiler-generated (which still shouldn't show up since they are private)?

Reply:
See my response here https://github.com/Azure/azure-sdk-for-net/pull/13522#discussion_r456128812